tests/testthat/test-bind_tables.R

```diff
@@ -0,0 +1,49 @@
+# Load initial variables
+token <- Sys.getenv("REDCAPTIDIER_LONGITUDINAL_API")
+redcap_uri <- Sys.getenv("REDCAP_URI")
+
+test_that("bind_tables works with no specifications", {
+
+  read_redcap_tidy(redcap_uri, token) %>%
+    bind_tables()
+
+  expect_true(exists("nonrepeated", envir = global_env()))
```

Comment:
Since the spec is that `bind_tables()` is that it `plop`s data frames into an environment, I think at least for the case in which the environment is the global environment, you could use `expect_s3_class(x, "data.frame")` here.

Reply:
Added 3 tests for the 3 dataframes expected 👍